src/files.js

```diff
@@ -90,6 +106,20 @@ function doUpload(cozy, data, method, path, options) {
   })
 }
 
+function addQueryString(path, querystring = {}) {
+  const pathurl = url.parse(path)
+  const query = qs.decode(pathurl.query)
+  Object.assign(query, querystring)
+  return pathurl.pathname + '?' + qs.encode(query)
+}
+
+async function sendMetadata(cozy, metadata) {
+  const result = await cozyFetchJSON(cozy, 'POST', '/files/upload/metadata', {
+    data: { type: 'io.cozy.files.metadata', attributes: metadata }
+  })
+  return result && result._id ? result._id : false
```

Comment:
style (if prettier let us), because such priorities are not clear at first sight

```suggestion
  return (result && result._id) ? result._id : false
```

Reply:
This can also be written like so:
```suggestion
  return (result && result._id) || false
```

Reply:
Prettier does not let us, sorry